x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -145,7 +156,7 @@
 import static org.elasticsearch.xpack.esql.core.type.DataType.UNSUPPORTED;
 import static org.elasticsearch.xpack.esql.core.type.DataType.VERSION;
 
-public final class EsqlFunctionRegistry extends FunctionRegistry {
+public class EsqlFunctionRegistry {
```

Comment:
nit: wondering if (maybe as a follow-up) we should remove the `Esql` prefix, ie. just `FunctionRegistry`

Reply:
Yeah. I think we can blast the `Esql` prefix on a bunch of stuff. I've sort of been ignoring that for now.